### PR TITLE
Tma 548 app txma integration

### PR DESCRIPTION
**Pull request by dthampi**
Added a new test to check the integration between APP and TxMA

### CODE REVIEW

File: tests/audit-tests/run-tests.sh [hunk 1]
```diff
@@ -54,14 +54,21 @@ if [ "$DRIVER" == "http://selenium-hub:4444/wd/hub" ]; then
 
 else
   echo "environment variable DRIVER exists but is not the selinium hubs service name. using the default localhost variant"
-  cd /
+#  cd /
```
Comment (zaydgds) [inline tests/audit-tests/run-tests.sh:57]: undo this comment as it is required for the pipeline to run

File: tests/audit-tests/run-tests.sh [hunk 2]
```diff
@@ -54,14 +54,21 @@ if [ "$DRIVER" == "http://selenium-hub:4444/wd/hub" ]; then
...
   echo "running selenium using chromedriver"
   export DRIVER=$DRIVER
-
 fi
+export CFN_AppURL="https://di-auth-stub-relying-party-staging-app.london.cloudapps.digital/"
```
Comment (zaydgds) [inline tests/audit-tests/run-tests.sh:61]: are these urls are the same for all the environments that run the tests?

**Review by zaydgds — CHANGES_REQUESTED** [review]
I cant comment on the test code themselves, probably best to get another tester to take a look at the test content itself

File: tests/audit-tests/run-tests.sh (hunk 2 above)
Comment (dthampi) [reply tests/audit-tests/run-tests.sh:61]: These tests run only in staging env

File: tests/audit-tests/run-tests.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (dthampi) [reply tests/audit-tests/run-tests.sh]: Done

**Review by PeterBrownGDS — APPROVED** [review]

**Review by zaydgds — APPROVED** [review]
looks good

**Pull request merged**